tutorials/snap/advanced-snap-usage.md

```diff
@@ -0,0 +1,1437 @@
+```
+
+This snap confirms that it can see the original (host) file system via `FILE SYSTEM: I see from /etc/os-release that I'm running on Ubuntu 16.04.2 LTS`. Also, the snap has access to everything (network, camera, and home directory), confirming the fact that it’s an augmented devmode snap.
+
+### The developer corner
+
+What does it mean for developers? If you want to snap a complex application and you don’t want to dive right away into making your application relocatable, you:
+  - can start with a classic snap, which should be easy to craft.
+  - then make your snap relocatable, and include all its dependencies. You can now turn on devmode! Remember, though, that you can’t publish it to the stable channel.
+  - finally, confine your application, turn it to strict mode and publish to the stable channel!
+
+This gives you a nice and progressive workflow.
+
+
+You can remove the 2 snaps now and reinstall one of them confined with:
+
+```bash
+$ snap remove chuck-norris-webserver-classic chuck-norris-webserver
+chuck-norris-webserver-classic removed
+chuck-norris-webserver removed
+$ snap install chuck-norris-webserver
+chuck-norris-webserver (stable) 1.0 from 'didrocks' installed
+```
+
+### Closing up
+
+A final note is that when you are connecting interfaces, those are only applied on command or service startup. You thus need to restart or relaunch a command to see those new capabilities apply. Launching a command or program is trivial. Restarting a service is a little bit more complex and that’s exactly what we are going to do next!
+
+
+## Availability, logs & states in snaps & services
+Duration: 6:00
+
+
+positive
+: Lost or starting from here?
+You just need to type in a terminal “`snap install chuck-norris-webserver`” as this is the he snap we will need.
+
+
+Before jumping right away into the logs, let’s look at snap states and availability on the system.
+
+### Snap availability on the system
+
+Once a snap is installed, this means:
+
+  - the snap commands are available in the user’s `$PATH`.
+  - the services included in the snap are all started. They are simply started/stopped on system startup and shutdown. Depending on their configuration, they can restart automatically in case of crashes.
+
+Not having a command running is easy: just don’t run it or stop it if it’s already started! But what about services?
+
+You can enable or disable snaps on the system. This means that a disabled snap won’t have any command accessible to any users on the system, nor services started at system startup. From a user perspective, it’s like the snap simply doesn’t exist. However, it’s still installed: binary files, assets and associated data are still present, and the snap is kept up to date. Making it available again is just a question of running the enable command. No extra download, nothing required!
+
+The chuck norris web server is running on port 80, so heading to http://localhost will lead you to:
```

Comment:
It seems a bit weird to me to do this. We took them out of a different tutorial..., but ok